codegen/generator/go/templates/format.go

```diff
@@ -33,7 +33,7 @@ func (f *FormatTypeFunc) FormatKindScalarBoolean(representation string) string {
 
 func (f *FormatTypeFunc) FormatKindScalarDefault(representation string, refName string, input bool) string {
 	if alias, ok := generator.CustomScalar[refName]; ok && input {
-		representation += "*" + alias
+		representation += alias
```

Comment:
Nit comment:

Wdyt about assigning directly `representation += refName` by default at the beginning and then check the condition `if alias, ok := generator.CustomScalar[refName]; ok && input {` ?

I am referring to this [guide](https://github.com/uber-go/guide/blob/master/style.md#unnecessary-else)

Reply:
The pattern you've mentioned can be used in case a variable gets **assigned** (`:=`). But in this case we **append** (`+=`) and thus I believe its not applicable.

Reply:
My bad, i didn't pay attention on the operators
Thanks for the clarification